developer/src/kmc-generate/test/fixtures/ldml-keyboard/sample/source/sample.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<keyboard3 xmlns="https://schemas.unicode.org/cldr/45/keyboard3" locale="en" conformsTo="45">
+  <info author="Sample Author" name="Sample Project"/>
+  <version number="1.0.0"/>
+
+  <keys>
+    <import base="cldr" path="45/keys-Zyyy-punctuation.xml"/>
+    <import base="cldr" path="45/keys-Zyyy-currency.xml"/>
```

Comment:
Is this the standard import for all LDML keyboards? (vs generator referencing files in a `45/` path)

Reply:
Good question @srl295

Reply:
Pinging @srl295 on this query

> Is this the standard import for all LDML keyboards? (vs generator referencing files in a 45/ path)
>
> ```
> <keys>
>   <import base="cldr" path="45/keys-Zyyy-punctuation.xml"/>
>   <import base="cldr" path="45/keys-Zyyy-currency.xml"/>
> ```